x/ccv/provider/keeper/grpc_query.go

```diff
@@ -150,24 +149,6 @@ func (k Keeper) QueryThrottleState(goCtx context.Context, req *types.QueryThrott
 	candidate := k.GetSlashMeterReplenishTimeCandidate(ctx) // always UTC
 	packets := []*types.ThrottledSlashPacket{}
```

Comment:
Yea my original intention here was to avoid breaking the query API. But you make a good point that this PR is already "query breaking", might as well make things clean -> https://github.com/cosmos/interchain-security/commit/1ceddcb356493162875d448b5fef2dfb07859a10

Reply:
To be consistent I should also remove this deprecated provider query too -> https://github.com/cosmos/interchain-security/commit/e228953c82039b3a7ac20343899af89cabc2afad